### ISSUE TITLE
Running trivy scan returns serialization error

### ISSUE THREAD

**Issue by niklev** [aquasecurity/intellij-trivy#11]
When running trivy (Version: 0.36.1)
trivy fs --security-checks=config,vuln,secret --severity=CRITICAL,HIGH,MEDIUM,LOW,UNKNOWN --format=json --output=C:\Users\<user>\AppData\Local\Temp\Trivy15924074954574361504.json C:/Users/<user>/IdeaProjects/<project>

Failed to deserialize the results file. No content to map due to end-of-input at [Source: (File); line: 1, column: 0]

Contents of the Trivy15924074954574361504.json is 0 bytes

**Comment by kahennig** [comment]
Hi niklev, 
I was facing the same issue with IDEA 2023.1.2 CE, Trivy findings explorer 1.5.0 and an old version of trivy. Once I upgraded to the latest version (0.43.1) the plugin started working fine. I am not related to the trivy, IDEA or the plugin, just a user and wondering if you managed to fix you issue.

**Comment by fischermatte** [comment]
still not working in 1.5.0 on intellji 2023.2. I have trivy 0.44.0 from snap

**Comment by imotsch** [comment]
Same problem here: IntelliJ 2023.2, trivy plugin 1.5.0, trivy_0.51.2_windows-64bit.
@Trivy when is this fixed?

**Comment by owenrumney** [comment]
I don't think this is an active issue so closing, but please reopen if I'm wrong
